packages/typir/src/features/operator.ts

```diff
@@ -92,10 +92,10 @@ export interface OperatorManager {
  * All operands are mandatory.
  */
 export class DefaultOperatorManager implements OperatorManager {
-    protected readonly typir: Typir;
+    protected readonly services: TypirServices;
 
-    constructor(typir: Typir) {
-        this.typir = typir;
+    constructor(services: TypirServices) {
```

Comment:
Not important, only a little inconsistency: Sometimes, the parameter for `TypirServices` is called `services` (like here), sometimes `typir` (e.g. for `DefaultTypeEquality`). That should be unified.
I am fine with `services`.

Reply:
I renamed all to `services`.